src/cpr_data_access/models.py

```diff
@@ -107,6 +109,18 @@ def _load_and_validate_metadata_csv(
     return metadata_df
 
 
+class KnowledgeBaseIDs(BaseModel):
+    """Store for knowledge base IDs."""
+
+    wikipedia_title: Optional[str]
+    wikidata_id: Optional[Annotated[str, constr(regex=r"^Q\d+$")]]  # type: ignore
```

Comment:
Note [`constr` will be deprecated in pydantic v3 so they advise to use `StringConstraints` instead](https://docs.pydantic.dev/latest/api/types/#pydantic.types.constr--__tabbed_1_2), but we're still on pydantic v1.x. We should maybe consider upgrading at some point.